percival/detector/detector.py

```diff
@@ -597,7 +597,7 @@ def setup_control(self):
         Creates a SystemCommand instance which can be used to send system commands to the hardware.
         """
         self._log.info("Carrier IP set as: %s", self._percival_params.carrier_ip)
-        self._txrx = TxRx(self._percival_params.carrier_ip)
+        self._txrx = TxRx(self._percival_params.carrier_ip, port=10001)
```

Comment:
Question: should the port be hard coded here?

Reply:
yes: If you disagree please say why.

Reply:
It was just a question about what happens if the port is changed on the detector firmware?
I guess it is expected never to be changed.

Reply:
You'd need to update the software; there is no command line option for it.